### DISCUSSION TITLE
[Feature request] Context menu in Flowbite

### DISCUSSION THREAD

**Discussion by ghost** [themesberg/flowbite#269] [Q&A] [answered]
Hi everyone! 

I would like to know if it would be possible to have a context menu in Flowbite. 

Some web applications use custom context menu for certain actions when the user uses the mouse. 

Since Flowbite is a visual framework, I thought it would be interesting to have this context menu feature to facilitate context menu development through a visual framework.

#### Idea
![image](https://user-images.githubusercontent.com/111701513/187854210-f4720b95-ecfb-4c6b-9400-f51259f1c53a.png)

#### source-code
https://codepen.io/Danny-Dasilva/pen/wvGaMxE

**Comment by zoltanszogyenyi** [comment, marked as answer]
Hey @codehangen,

I've actually been thinking that it would be nice to introduce this component to Flowbite.

We will most likely first design this in Figma and then release it as an open-source component.

@robert1508 check this out.

Cheers,
Zoltan

**Reply by ghost** [reply]
Hi zoltanszogyenyi.

> Hey @codehangen,
> I've actually been thinking that it would be nice to introduce this component to Flowbite.
> We will most likely first design this in Figma and then release it as an open-source component.
> @robert1508 check this out.
> Cheers,
> Zoltan

thank you for comment.

**Comment by bbgrk** [comment]
Hey. Any update on this one? Thanks

**Comment by magic-madrigal** [comment]
Any update on this one?
